src/main/java/javaClasses/homework_5/Booking/Stoppable.java

```diff
@@ -0,0 +1,9 @@
+package javaClasses.homework_5.Booking;
+
+
+public interface Stoppable {
+
+    default String makingStop() {
+        return null;
```

Comment:
The idea behind default method, is to provide some common implementation for method users, so they don't have their own once. Please change this to print any message instead of returning null.

Reply:
I have added the message.